main.go

```diff
@@ -17,21 +17,27 @@ import (
 
 const (
 	baseURL                   = "https://api.bitrise.io/v0.1"
-	buildTypeTag              = "tag"
-	buildTypePush             = "push"
-	buildTypeManual           = "manual"
-	buildTypePullRequest      = "pull-request"
 	envKeyPreviousBuildStatus = "PREVIOUS_BUILD_STATUS"
 	envKeyBuildStatusChanged  = "BUILD_STATUS_CHANGED"
 	statusTextSuccessfulBuild = "success"
 )
 
+type buildType int
```

Comment:
I would move this block to a new internal package, so the client side would be: `buildtype.Push` etc..